pountzas/math-magicians-react
# Math Magicians: tests

## Math Magicians: tests

- Write unit tests with React Testing Library
- Write unit tests with snapshots with Jest
- Write unit tests for the files operate.js and calculate.js using Jest
- Create unit tests for all React components
- Use Jest snapshots to test the components
- Use React Testing Library to simulate user interaction
- All tests should pass
